jest.config.js

```diff
@@ -31,7 +31,7 @@ module.exports = {
   moduleNameMapper: pathNames,
   roots: ['<rootDir>/tests', '<rootDir>/src'],
   preset: 'ts-jest',
-  testRegex: '/tests/.*.test.tsx?$',
+  testMatch: ['**/?(*.)+(spec|test).(js|ts|tsx)'],
```

Comment:
Are you sure we want to allow both `spec` and `test` rather than forcing one or the other?

Reply:
Yeah could get messy if we use both, +1 for using one 1️⃣